doc/mcuboot/index-ncs.rst

```diff
@@ -0,0 +1,68 @@
+.. _mcuboot_index_ncs:
+
+MCUboot
+#######
+
+.. contents::
+   :local:
+   :depth: 2
+
+MCUboot is a secure bootloader for microcontrollers.
+
+Overview
+********
+
+The goal of MCUboot is to define a common infrastructure for the bootloader, the system flash layout on microcontroller systems, and to provide a secure bootloader that enables easy software upgrade.
```

Comment:
```suggestion
MCUboot defines a common infrastructure for the bootloader and the system flash layout on microcontroller systems, and provides a secure bootloader that enables easy software upgrade.
```
`the goal of` is not needed in my opinion.

Reply:
fixed